python/tvm/contrib/hexagon/build.py

```diff
@@ -110,6 +110,7 @@ def __init__(self, rpc_info: dict, workspace: Union[str, pathlib.Path] = None):
         }
         self._rpc_info.update(rpc_info)
         self._workspace = self._create_workspace(workspace)
+        self._device_key = self.HEXAGON_REMOTE_DEVICE_KEY
```

Comment:
wouldn't this always set the key to be `self.HEXAGON_REMOTE_DEVICE_KEY`? My understanding is that the goal is to be able to set the `key` to distinguish RPC server between different tests/runs.

Reply:
The idea is to get the PID of the process that starts the server.  The launcher itself may be created once ahead of time, and then we'd have the same PID for every server.  Since the base class only has an abstract method `start_server`, the appending of the PID is done in the actual implementations.  I'm setting the member `_device_key`, here so that it exists in the object.

At the moment, the PID is only appended for the simulator.  Do you think we should do that for Android as well?

Reply:
This is okay as it is. Not sure if this could solve some of the existing issues with test flakiness on Android. We can consider it later.